src/studio/src/designer/backend/Controllers/TextsController.cs

```diff
@@ -81,14 +81,26 @@ public async Task<ActionResult> Put(string org, string repo, string languageCode
         [HttpGet]
         [Produces("application/json")]
         [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
         [Route("{languageCode}")]
         public async Task<ActionResult<Dictionary<string, string>>> Get(string org, string repo, [FromRoute] string languageCode)
         {
             string developer = AuthenticationHelper.GetDeveloperUserName(HttpContext);
 
-            Dictionary<string, string> text = await _textsService.GetTexts(org, repo, developer, languageCode);
-
-            return Ok(text);
+            try
+            {
+                Dictionary<string, string> text = await _textsService.GetTexts(org, repo, developer, languageCode);
+                return Ok(text);
+            }
+            catch (IOException)
+            {
+                return new ObjectResult(new { errorMessage = "The texts you are trying to find does not exist." }) { StatusCode = 404 };
```

Comment:
Minor, but on both of these it would be good to add information about what the filename actually is. "The format of the nb.texts.json file is invalid."